components/filter/SubregionForm.tsx

```diff
@@ -63,19 +63,24 @@ const SubregionCheckbox = ({ region, subregion }: SubregionCheckboxProps) => {
   );
 };
 
-const SubregionForm = () => {
+interface SubregionFormProps {
+  region: string;
+}
+
+const SubregionForm = ({ region }: SubregionFormProps) => {
   return (
-    <Stack p="18px 24px" spacing="12px" divider={<StackDivider borderColor="gray.200" />}>
-      {Object.keys(regions).map(function (region) {
-        return [
-          <Stack key="subregions">
-            {regions[region].map((subregion): ReactElement => {
-              return <SubregionCheckbox key={subregion} region={region} subregion={subregion} />;
-            })}
-          </Stack>,
-        ];
-      })}
-    </Stack>
+    // I realise this is a very hacky way of getting the alignment for the tree to work.
+    // Please suggest another method if possible.
+    <HStack>
+      <Box width="15%" />
+      <Box width="85%">
+        <VStack width="inherit" align="left">
+          {regions[region].map((subregion): ReactElement => {
+            return <SubregionCheckbox key={subregion} region={region} subregion={subregion} />;
+          })}
+        </VStack>
```

Comment:
You can remove the HStack and the two Box elements, then just set the left padding or margin (e.g. pl="10px") on VStack.